### DISCUSSION TITLE
Is it possible to add color to text in Rich text editor?

### DISCUSSION THREAD

**Discussion by GabriFila** [mantinedev/mantine#1090] [Q&A] [answered]
Hello, I've found Mantine Rich text editor which I found very useful and fast to implement.

I would like to know if there is a possibility to let the user color text inside the editor and how difficult would it be to implement. I would like it to be a picker in the toolbar.

Thank you everyone!

**Comment by rtivital** [comment, marked as answer]
No, currently it is not supported

**Comment by GabriFila** [comment]
Hi @rtivital ! Do you have any updates on this?

Thank you

**Reply by rtivital** [reply]
No, currently it is not supported
